test/test_torch.py

```diff
@@ -6977,15 +6977,64 @@ def compare_strides(s1, s2, div):
 
     @skipMeta
     @dtypes(*torch.testing.get_all_dtypes())
-    def test_dlpack_conversion(self, device, dtype):
-        # DLpack does not explicitly support bool
+    def test_dlpack_capsule_conversion(self, device, dtype):
+        # DLpack does not explicitly support bool (xref dmlc/dlpack#75)
         # It does it through uint8 type
-        if dtype is torch.bool:
+        if dtype is torch.bool or 'xla' in device:
```

Comment:
Don't skip the test like this. Use the @onlyOnCPUAndCUDA decorator to run only on CPU and CUDA devices. Remove torch.bool from the set of dtypes passed to the `@dtypes` decorator if the test doesn't want to test bool.

However, what about asserting an error is thrown for bool tensors?

Reply:
Done, I added a test just for bool tensors :)